registry/accessing-the-registry.adoc

```diff
@@ -49,8 +49,10 @@ include::modules/registry-viewing-logs.adoc[leveloffset=+1]
 
 include::modules/registry-accessing-metrics.adoc[leveloffset=+1]
 
-.Additional resources
+include::modules/
 
+== Additional resources
```

Comment:
You could also potentially add a empty line after the heading for readability within the assembly file. But, it makes no difference to the build so it is not essential.